lib/render/vp2ShaderFragments/Float4ToFloat3.xml

```diff
@@ -1,58 +1,58 @@
-<fragment uiName="float4ToFloat4" name="float4ToFloat4" type="plumbing" class="ShadeFragment" version="1.0" feature_level="0">
+<fragment uiName="Float4ToFloat3" name="Float4ToFloat3" type="plumbing" class="ShadeFragment" version="1.0" feature_level="0">
```

Comment:
git history is messed up due to file renaming. But the final xml files are correct.